### PR TITLE
EIP-2569: move from "stagnant" to "draft"

### PR DESCRIPTION
**Pull request by lemontreeran**
- Move EIP-2569 from "stagnant" to "draft" to continue the review process
- Add new author to follow up the review process

### CODE REVIEW

**Review by zhous — APPROVED** [review]
OK, let's do it!

**Comment by lemontreeran** [comment]
> @lightclient is the most likely person to review this (I only handle Core EIPs at the moment). As part of that they may give some feedback on the structure (like Simple Summary section has been deprecated in favor of a `description` header field) before approving, but those notes will likely be relatively limited since this EIP presumably already went through the first tier of editorial process. Just be ready to respond to any feedback you get quickly to avoid delays. 😄

Thanks so much @MicahZoltu ! Will follow up for any feedback ASAP.

**Comment by lemontreeran** [comment]
Hello @lightclient , thanks for your review. I have updated the structure based on your feedback. Please help and have another look.

File: EIPS/eip-2569.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
 ---
 eip: 2569
 title: Saving and Displaying Image Onchain for Universal Tokens
-author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous)
+author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous), Ran Xing (@lemontreeran)
+description: A set of interfaces to save an SVG image in Ethereum, and to retrieve the image file from Ethereum for universal tokens.
```
Comment (axic) [inline EIPS/eip-2569.md:5]: Please swap these two lines (description comes first).

Comment (lemontreeran) [reply EIPS/eip-2569.md:4]: Hi @axic , Change is made according to your suggestion. Thanks a lot for your approval.

**Comment by lemontreeran** [comment]
Hi @axic and @lightclient , Thanks for reviewing this PR. Any question, please let us know. Thanks so much🙏

File: EIPS/eip-2569.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
 ---
 eip: 2569
 title: Saving and Displaying Image Onchain for Universal Tokens
-author: Hua Zhang (@dgczhh), Yuefei Tan (@whtyfhas), Derek Zhou (@zhous)
+description: A set of interfaces to save an SVG image in Ethereum, and to retrieve the image file from Ethereum for universal tokens.
```
Comment (lightclient) [inline EIPS/eip-2569.md:4]:
```suggestion
description: A set of interfaces to save an SVG image in Ethereum, and to retrieve the image file from Ethereum for universal tokens.
```

**Review by lightclient — APPROVED** [review]

**Pull request merged**